jawn-core/src/main/java/net/javapla/jawn/core/templates/ContentTemplateLoader.java

```diff
@@ -12,11 +14,14 @@
 	private final String realPath;
 	private final TemplateEngine.TemplateRenderEngine<T> engine;
 	private final String templateSuffix;
+	private final int templateSuffixLengthToRemove;
+	private final Map<String,String> layoutMap = new HashMap<>();
```

Comment:
As this is an internal reference, I would like the `HashMap` to keep being a `HashMap`

I try to have a way of thinking, that only APIs (or public methods) should return interfaces

Reply:
Changed to HashMap :)